import { BaseException } from "./base.exception";

export class RecursoNaoEncontradoException extends BaseException {
  constructor() {
    super("Recurso buscado n√£o encontrado.");
  }

  statusCode = 404;
}
